New Sample: MyShifts

siddharth-vaghasia opened the pull request:
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| New sample?     |  yes                               |


## What's in this Pull Request?
This adaptive card extension sample demonstrates the ability to show Upcoming Shifts based on the end user's preference configurable settings to select Team and store the configuration in the OneDrive application folder.

dcashpeterson commented:
@siddharth-vaghasia Thanks for the new sample. I will get it reviewed and processed as soon as I can.

dcashpeterson commented:
@siddharth-vaghasia I'm not sure what's going on when I try to build this but I am getting a bunch of errors. Can you take a look at it again and let me know. Also, it's using the beta for 1.14 so would you mind change it to use the RC?

Here is the error I am getting.
Error - [tslint] [{"endPosition":{"character":7,"line":57,"position":1662},"failure":"Missing semicolon","fix":{"innerStart":1662,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/cardView/CardView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":7,"line":57,"position":1662}},{"endPosition":{"character":7,"line":65,"position":2062},"failure":"Missing semicolon","fix":{"innerStart":2062,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/cardView/CardView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":7,"line":65,"position":2062}},{"endPosition":{"character":7,"line":71,"position":2353},"failure":"Missing semicolon","fix":{"innerStart":2353,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/cardView/CardView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":7,"line":71,"position":2353}},{"endPosition":{"character":11,"line":79,"position":2553},"failure":"Missing semicolon","fix":{"innerStart":2553,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/cardView/CardView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":11,"line":79,"position":2553}},{"endPosition":{"character":13,"line":85,"position":2687},"failure":"Missing semicolon","fix":{"innerStart":2687,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/cardView/CardView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":13,"line":85,"position":2687}},{"endPosition":{"character":40,"line":31,"position":1105},"failure":"Missing semicolon","fix":{"innerStart":1105,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/quickView/ConfigureTeamView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":40,"line":31,"position":1105}},{"endPosition":{"character":46,"line":33,"position":1162},"failure":"Missing semicolon","fix":{"innerStart":1162,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/quickView/ConfigureTeamView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":46,"line":33,"position":1162}},{"endPosition":{"character":103,"line":59,"position":2258},"failure":"Missing semicolon","fix":{"innerStart":2258,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/quickView/ConfigureTeamView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":103,"line":59,"position":2258}},{"endPosition":{"character":110,"line":61,"position":2417},"failure":"Missing semicolon","fix":{"innerStart":2417,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/quickView/ConfigureTeamView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":110,"line":61,"position":2417}},{"endPosition":{"character":44,"line":66,"position":2587},"failure":"Missing semicolon","fix":{"innerStart":2587,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/quickView/ConfigureTeamView.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":44,"line":66,"position":2587}},{"endPosition":{"character":40,"line":31,"position":1083},"failure":"Missing semicolon","fix":{"innerStart":1083,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/quickView/SubmitMessage.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":40,"line":31,"position":1083}},{"endPosition":{"character":12,"line":24,"position":881},"failure":"Properties should be separated by semicolons","fix":{"innerStart":880,"innerLength":1,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":11,"line":24,"position":880}},{"endPosition":{"character":20,"line":61,"position":2142},"failure":"Missing semicolon","fix":{"innerStart":2142,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":20,"line":61,"position":2142}},{"endPosition":{"character":47,"line":82,"position":2555},"failure":"Missing semicolon","fix":{"innerStart":2555,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":47,"line":82,"position":2555}},{"endPosition":{"character":44,"line":85,"position":2674},"failure":"Missing semicolon","fix":{"innerStart":2674,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":44,"line":85,"position":2674}},{"endPosition":{"character":120,"line":92,"position":3014},"failure":"Missing semicolon","fix":{"innerStart":3014,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":120,"line":92,"position":3014}},{"endPosition":{"character":37,"line":106,"position":3586},"failure":"Missing semicolon","fix":{"innerStart":3586,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":37,"line":106,"position":3586}},{"endPosition":{"character":124,"line":111,"position":3890},"failure":"Missing semicolon","fix":{"innerStart":3890,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":124,"line":111,"position":3890}},{"endPosition":{"character":97,"line":152,"position":5289},"failure":"Missing semicolon","fix":{"innerStart":5289,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":97,"line":152,"position":5289}},{"endPosition":{"character":104,"line":154,"position":5436},"failure":"Missing semicolon","fix":{"innerStart":5436,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/sa
Error - [tslint] mples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":104,"line":154,"position":5436}},{"endPosition":{"character":38,"line":159,"position":5576},"failure":"Missing semicolon","fix":{"innerStart":5576,"innerLength":0,"innerText":";"},"name":"/Users/xxx/Documents/Projects/sp-dev-fx-aces-dev/samples/PrimaryTextCard-MyShifts/src/adaptiveCardExtensions/myShifts/MyShiftsAdaptiveCardExtension.ts","ruleName":"semicolon","ruleSeverity":"ERROR","startPosition":{"character":38,"line":159,"position":5576}}]

siddharth-vaghasia commented:
@dcashpeterson - thanks for reviewing, the errors seem to be about semicolon missings which would have been strict tslint rules at your side, and for me they were warnings, nevertheless I have fixed it....  I have also updated the SVG of SPFx version to depict beta version, at the time of submission the beta version shied was not thier for beta...
